agent/agentlocal/agent_local.go

```diff
@@ -61,9 +61,15 @@ const (
 	serverZipFile   = "pmm-agent.log"
 )
 
+// configGetReloader allows for getting and reloading a config.
```

Comment:
```suggestion
// configGetReloader allows to get and reload a config.
```
nit: I was looking for simplicity :)